Makefile

```diff
@@ -188,7 +188,7 @@ CFLAGS_PRE:=\
         -fdata-sections
 
 CFLAGS:=$(FLAGS_CM4) $(CFLAGS_PRE)
-CPPFLAGS:=-fno-exceptions
+CXXFLAGS:=-fno-exceptions
```

Comment:
It was easy to do in 2 commits, and I think it makes sense to separate the change into two commits in order to keep functional changes separate from cosmetic ones; it's easier to see that "nothing is up my sleeve".